package.json

```diff
@@ -42,7 +45,7 @@
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.12"
   },
-  "prettier" : {
+  "prettier": {
     "singleQuote": true
   }
```

Comment:
I would probably move this to a `.prettierrc.yml` file even though it's only one line.

Reply:
Well, wrong PR, but can do.